### ISSUE TITLE
Tooltip ADA review

### ISSUE THREAD

**Issue by parsakhan21** [jpmorganchase/salt-ds#1163]

**Comment by parsakhan21** [comment]
March 24 - in progress ; follow up with Alina

**Comment by joshwooding** [comment]
March 29 - Waiting for second review by @maj-stella

**Comment by maj-stella** [comment]
Point 1 and Point 2 have been fixed; however the arrow tooltip is still misplaced.

Also the same issue ‘Tooltip is obscuring buttons’ appears in the ‘Placement’ example. Please see the screenshot below.

![Image](https://user-images.githubusercontent.com/112501956/228632044-6fba4424-5de3-4488-be04-a1095d1c791f.png)



Additionally the `<br>` elements need to be removed as they were used for layout; They are semantic elements and screen reader announces them as ‘blank’. It’s very distracting to listen ‘blank, blank’ for screen reader users.

![Image](https://user-images.githubusercontent.com/112501956/228632155-5851926a-6b1b-4238-af56-a4397471c54d.png)

**Comment by alycrys** [comment]
All issues should be fixed now. @maj-stella can you please confirm?

**Comment by maj-stella** [comment]
Url tested: https://628e27029fcf95004a62cafd-ujapvsvjkq.chromatic.com/?path=/story/core-tooltip--status

The zoom in bug still exist on MacOS Monterey in Chrome v Version 111.0.5563.110 (Official Build) (x86_64)

![Image](https://user-images.githubusercontent.com/112501956/229176219-b460808c-b2ce-4707-95eb-66ddb3d8bc59.png)

**Comment by alycrys** [comment]
Unfortunately I cannot reproduce this issue anymore.
Can I suggest we live with this? This is an edge case and it doesn't affect the user journey or accessibility.
